internal/storage/sbom_schema.go

```diff
@@ -0,0 +1,20 @@
+package storage
+
+const CreateSBOMTableSQL = `
+CREATE TABLE IF NOT EXISTS sboms (
+    id INTEGER PRIMARY KEY,
+    name VARCHAR(253) NOT NULL,
+    namespace VARCHAR(253) NOT NULL,
+    object TEXT NOT NULL,
+    UNIQUE(name, namespace)
+);
```

Comment:
Based on what I've seen below, we put the json encoded object (the whole CRD) into the `object` column.

I think we have to extract the labels from the object and store them into dedicated columns. Users can do `GET`, `DELETE`, `WATCH` operations using label selectors

Reply:
`GET` and `DELETE` operations do not support label selectors directly.

As per the `GetList`, which is also used by the `WATCH`, we are getting all the objects under a certain key, and then we are applying the label predicate programmatically. See https://github.com/fabriziosestito/sbombastic/blob/e0fd7b22decd0fe97054d45f6d958b31ffb14570/internal/storage/sbom_store.go#L294

I've tracked this as debt here  #16 and as a feature here: #15